x-pack/legacy/plugins/siem/server/lib/detection_engine/alerts/types.ts

```diff
@@ -38,11 +40,13 @@ export interface SignalAlertParams {
   savedId: string | undefined;
   severity: string;
   size: number | undefined;
+  tags: string[];
   to: string;
   type: 'filter' | 'query' | 'saved_query';
 }
 
 export type SignalAlertParamsRest = Omit<SignalAlertParams, 'maxSignals' | 'saved_id'> & {
```

Comment:
Do you need to include the camelCase `falsePositives` in the omit as well?

e.g. 
```suggestion
export type SignalAlertParamsRest = Omit<SignalAlertParams, 'falsePositives' | 'maxSignals' | 'saved_id'> & {
```

Reply:
You are correct! Thank you, fixed and pushed.